bitseq/sequence.go

```diff
@@ -262,19 +269,11 @@ func (h *Handle) runConsistencyCheck() bool {
 // It looks for a corruption signature that may happen in docker 1.9.0 and 1.9.1.
 func (h *Handle) CheckConsistency() error {
 	for {
-		h.Lock()
-		store := h.store
-		h.Unlock()
-
-		if store != nil {
-			if err := store.GetObject(datastore.Key(h.Key()...), h); err != nil && err != datastore.ErrKeyNotFound {
-				return err
-			}
+		if err := h.Refresh(); err != nil {
```

Comment:
do you need to lock around the Refresh? (Refresh inside does not get any lock, so h.store can become nil?)